src/OpenTelemetry/Metrics/MeterProviderSdk.cs

```diff
@@ -157,7 +157,18 @@ internal sealed class MeterProviderSdk : MeterProvider
                         var metricStreamConfigs = new List<MetricStreamConfiguration>(viewConfigCount);
                         foreach (var viewConfig in this.viewConfigs)
                         {
-                            var metricStreamConfig = viewConfig(instrument);
+                            MetricStreamConfiguration metricStreamConfig = null;
+
+                            try
+                            {
+                                metricStreamConfig = viewConfig(instrument);
+                            }
+                            catch (ArgumentException ex)
+                            {
+                                OpenTelemetrySdkEventSource.Log.MetricInstrumentIgnored(instrument.Name, instrument.Meter.Name, "Applying View Configuration failed with error:" + ex.Message, "Fix the view configuration.");
+                                return;
```

Comment:
Just to confirm - we want to proceed (with a warning/error msg) rather than rethrow?

Reply:
Yes. This is instrument creation path, and it can occur anytime, so we are following the general principle of not punishing user after SDK is initialized successfully.

Reply:
I see, thanks!